content/commands.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Commands
+order: 46
+description: A reference for Galaxy's commands
+---
+
+<h2 id="deployment">Deployment</h2>
+
+The deployment command is as follows:
+
+`DEPLOY_HOSTNAME=[region] meteor deploy [hostname] --settings [path-to-settings-file]`
+
+- `region` should be us-east-1.galaxy.meteor.com for the US region and eu-west-1.galaxy.meteor.com for the EU region
```

Comment:
Honestly you really don't have to specify DEPLOY_HOSTNAME at all, unless one of these conditions is true:

- You are running a version of Meteor older than 1.3.3
- You are deploying to something other than default US production (most obviously EU, but also ties or staging) *AND* you haven't set up DNS for the hostname you're typing to point to Galaxy

I put a bunch of work into making this magically work but our docs all continue to tell people to do it the ugly hard way :(